app/views/nsm/claims/_claim_summary.html.erb

```diff
@@ -1,17 +1,30 @@
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
-      <% if claim.display_state?  %>
-        <p class="govuk-!-margin-bottom-2">
-          <%= govuk_tag(text: t(".status_list.#{'granted'}"), colour: accessed_colour('granted')) %>
```

Comment:
This was my bad - I hard-coded the state in an earlier PR while testing something out and didn't notice that I forgot to take it out again!